controller_manager:
  ros__parameters:
    update_rate: 50

diff_drive_controller:
  ros__parameters:
    type: diff_drive_controller/DiffDriveController
    left_wheel_names: ["left_wheel_back_joint", "left_wheel_front_joint"]
    right_wheel_names: ["right_wheel_back_joint", "right_wheel_front_joint"]
    wheel_separation: 0.26
    wheel_radius: 0.05
    use_stamped_vel: false
    publish_rate: 50.0
    cmd_vel_timeout: 0.5
    enable_odom_tf: true
    odom_frame_id: odom
    base_frame_id: base_link
    cmd_vel_topic: cmd_vel
    odom_topic: odom

joint_state_broadcaster:
  ros__parameters:
    type: joint_state_broadcaster/JointStateBroadcaster

